crates/pcb/src/mcp.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::Result;
+use clap::Args;
+use pcb_mcp::ResourceInfo;
+
+#[derive(Args, Debug)]
+pub struct McpArgs {}
+
+pub fn execute(_args: McpArgs) -> Result<()> {
+    let mut tools = vec![];
+
+    #[cfg(feature = "api")]
+    tools.extend(pcb_diode_api::mcp::tools());
+
+    // Point to public Zener documentation (client fetches directly)
+    let resources = vec![ResourceInfo {
```

Comment:
The reason we have this as a resource and a tool is that amp doesn't support resources?

Reply:
yep